python/dalex/dalex/_arena/_plot_container.py

```diff
@@ -0,0 +1,61 @@
+from .params import ModelParam, DatasetParam, VariableParam, ObservationParam
+
+class PlotContainer:
+    def __init__(self, arena):
+        if type(arena).__name__ != 'Arena' or type(arena).__module__ != 'dalex._arena.object':
```

Comment:
This can be `not isinstance(arena, dx._arena.object.Arena)`

Reply:
I tried using isinstance, but end up with circular dependencies.